addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -385,6 +385,18 @@ odoo.define('point_of_sale.Chrome', function(require) {
         _replaceCrashmanager() {
             var self = this;
             CrashManager.include({
+                show_warning: function (error) {
+                    if (self.env.pos) {
+                        // self == this component
+                        self.showPopup('ErrorPopup', {
+                            title: error.message,
```

Comment:
```suggestion
                            title: error.data.title.toString(),
```
I'm tested this and I believe we can do this suggestion.

I looked into the `CrashManager` code and it seems that the field `title` the result of `_lt`. So calling the `toString` will actually return the translated value. What do you think?

Reply:
@done !